### DISCUSSION TITLE
Better Logs when using `gradle run`

### DISCUSSION THREAD

**Discussion by rohitkg98** [micronaut-projects/micronaut-core#3957] [Ideas] [answered]
When running the application using `gradle run`, most of the logs are about Application Context initialization.

Can we have better formatting of these logs and make them concise and clear?

For example:
In Liquibase Migration Runner, we can make it so it only shows the Migration ID that got applied rather than all the queries that got fired.

In case of failing to initialize a Hibernate Bean, just a concise message that it failed, rather than the whole stacktrace.

There are many more places where an improvement can be made. I would like the community's views on this. I am happy to contribute to this effort with guidance in the right direction.

**Comment by graemerocher** [comment]
Thanks for the feedback. I encourage you to create individual issues in the different modules and send PRs to address problematic logging and stack traces those can be merged individually and of course contributions are welcome. 

Many of these will not be to core, for example for Hibernate you would send to https://github.com/micronaut-projects/micronaut-sql/

**Comment by alvarosanchez** [comment, marked as answer]
Also, it depends on how you configure Logback. By default, `gradle run` will show a single line when the server is running.

You should avoid configuring the root logger to a high level like debug or trace, and instead configure specific loggers for the packages you are interested into. In general, we produce debug and/or trace logging where applicable, but if you see fit for adding/removing messages, please raise issues in the corresponding projects.
